### PR TITLE
fix: [ANDROAPP-1670] scan only options of optionSet

### PR DESCRIPTION
**Pull request by mmmateos**
https://jira.dhis2.org/browse/ANDROAPP-1670

### CODE REVIEW

File: app/src/main/java/org/dhis2/usescases/qrScanner/ScanActivity.kt [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        mScannerView.startCamera()
+    }
+
+    override fun onRequestPermissionsResult(
+        requestCode: Int,
+        permissions: Array<String>,
+        grantResults: IntArray
+    ) {
+        when (requestCode) {
+            101 -> {
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/qrScanner/ScanActivity.kt]: can you use a constant?

File: app/src/main/java/org/dhis2/usescases/qrScanner/ScanActivity.kt [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        optionSetUid = intent.getStringExtra(Constants.OPTION_SET)
+        (applicationContext as App)
+            .userComponent()
+            ?.plus(ScanModule(optionSetUid))
+            ?.inject(this)
+
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_qr)
```
Comment (ferdyrod) [inline app/src/main/java/org/dhis2/usescases/qrScanner/ScanActivity.kt]: Maybe it is a good idea to also change the name of the layout to correspond to the activity's name? What do you think?

File: app/src/main/java/org/dhis2/usescases/qrScanner/ScanActivity.kt (hunk 1 above)
Comment (mmmateos) [reply app/src/main/java/org/dhis2/usescases/qrScanner/ScanActivity.kt]: Sure!

File: app/src/main/java/org/dhis2/usescases/qrScanner/ScanActivity.kt (hunk 2 above)
Comment (mmmateos) [reply app/src/main/java/org/dhis2/usescases/qrScanner/ScanActivity.kt]: You are right

**Pull request merged**